### ISSUE TITLE
Does not boot with Velocity 3.0.0.

### ISSUE THREAD

**Issue by Tydium** [McMackety/velocity-queue#19]
Tried to boot the plugin via velocity 3.0.0 and it didn't initiate.

**Comment by worldofrice** [comment]
Can't create plugin velocityqueue
velocity 3.0.0

**Comment by McMackety** [comment]
Yeah, there are other queue plugins being made for Velocity 3.0.0, I would recommend you use those. I'm no longer doing much in Minecraft and won't continue to maintain this.
